06.bowling_object/spec/game_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require_relative '../class/game'
+
+describe Game do
+  describe 'play and score' do
+    subject do
+      game = Game.new
+      game.play(marks)
+      game.score
+    end
+
+    context 'when marks has strike and spare and no mark' do
+      let(:marks) { %w[6 3 9 0 0 3 8 2 7 3 X 9 1 8 0 X 6 4 5] }
+      it 'return 139' do
+        is_expected.to eq 139
+      end
```

Comment:
```suggestion
      it { is_expected.to eq 139 }
```

RSpecの設計思想としては、subjectを使う場合は「英語っぽく読めるようにする」という使い方を想定していますね。